### PR TITLE
VIH-11199 Clear audio alert if video closed

### PR DESCRIPTION
**Pull request by will-craig**
### Jira link
https://tools.hmcts.net/jira/browse/VIH-11199

### CODE REVIEW

**Review by marcogagliardi — APPROVED** [review]
LG2M

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts [hunk 1]
```diff
@@ -133,6 +133,10 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
         return !this.hearing.isClosed() && !this.hearing.isInSession();
     }
 
+    videoClosedExt() {
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts:136]: Did you mean to leave a method that just logs?

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts [hunk 2]
```diff
@@ -1307,6 +1307,10 @@ export abstract class WaitingRoomBaseDirective implements AfterContentChecked {
         this.streamInMain = !this.streamInMain;
     }
 
+    videoClosedExt() {
+        this.logger.debug(`${this.loggerPrefix} - video closed`);
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts:1311]: Did you mean to have a method that just logs a single line?

Comment (will-craig) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts:1311]: Yep, it's to extend the functionality of the setShowVideo function, so I can override it in the child - judge waiting room - class

**Review by komaldabhi1509 — APPROVED** [review]

**Pull request merged**